### PR TITLE
fixes #477

### LINKED ISSUE #477: [Bug] Floor building with PR#267
This merge #267 introduces a bug when building the floor. When placing a row of flooring south from the starting area. The jobs start at the end of the column but are unreachable so they throw an error: `ArgumentNullException: Argument cannot be null.`  in `Character` line 177 and spawns in anyway.

1b
2b

### PR DESCRIPTION
**Pull request by Randshot**

### CODE REVIEW

File: Assets/Scripts/Models/Character.cs [hunk 1]
```diff
@@ -174,8 +174,8 @@ void GetNewJob()
         if (myJob.adjacent)
         {
             IEnumerable<Tile> reversed = pathAStar.Reverse();
-            reversed = reversed.Skip(1);
-            pathAStar = new Path_AStar(new Queue<Tile>(reversed.Reverse()));
+            reversed = reversed == null ? null : reversed.Skip(1);
+            pathAStar = reversed == null ? pathAStar : new Path_AStar(new Queue<Tile>(reversed.Reverse()));
```
Comment (vogonistic) [inline Assets/Scripts/Models/Character.cs]: These two lines would have been cleaner with an if-statement.

**Comment by vogonistic** [comment]
What does the change in `Path_TileGraph.cs` do to this bug?

**Comment by vogonistic** [comment]
I get 4 errors each time it tries and fails to find the path to a job with this change.

**Comment by vogonistic** [comment]
Are you on Discord? We've setup an unofficial here: https://discord.gg/68hkpSA

**Comment by Randshot** [comment]
Yeah I do have Discord.

**Comment by vogonistic** [comment]
Fixed in #481, closing.

**Pull request closed without merging**